tests/messageRoute.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('message route', async ({ page }) => {
+  await page.goto('http://localhost:3000/home-page');
+  await page.getByRole('link', { name: 'Image of Red Nike Trainers' }).click();
```

Comment:
Watch out for this locator. I don't have the red shoes on my `/home-page` when I log in so this might not always be obtainable, specially since the section highlighting items on the home page is for the most recently added.

The test should target the first `link` within the `div` with class 'lastItems' probably.

Reply:
Updated this to     
```await expect(
      page.locator('.conversation-height > div:nth-child(3)')
    ).toBeVisible();```